# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/ASpdySession.h
# Commit: 496ec0c5a60f
# Full Hash: 496ec0c5a60fdb86b233d036dd4747c477fc33ca
# Author: Dragana Damjanovic <dd.mozilla@gmail.com>
# Date: 2019-10-31 09:53:09
# Regressor Bug: 1581637
# File Overlap Count: 1
# Description:
#   Bug 1581637 - Part 8 - Add Http3Session/Http3Stream. r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D46652
# ==============================================================================

diff -r 63b7f1ff1714 -r 496ec0c5a60f netwerk/protocol/http/ASpdySession.h
--- a/netwerk/protocol/http/ASpdySession.h	Wed Oct 30 20:54:21 2019 +0000
+++ b/netwerk/protocol/http/ASpdySession.h	Wed Oct 30 21:09:09 2019 +0000
@@ -63,7 +63,7 @@
   // soft errors are errors that terminate a stream without terminating the
   // connection. In general non-network errors are stream errors as well
   // as network specific items like cancels.
-  bool SoftStreamError(nsresult code) {
+  static bool SoftStreamError(nsresult code) {
     if (NS_SUCCEEDED(code) || code == NS_BASE_STREAM_WOULD_BLOCK) {
       return false;
     }